README.md

```diff
@@ -0,0 +1,82 @@
+# openpaas-front
+
+Repository that contains recipes to build & publish OpenPaaS frontends :
+* Inbox
+* Mailto
+* Account
+* Contacts
+* Calendar
+* Calendar Public
+* Admin
+
+Every frontends is built separately and added in Docker image to be served by Nginx.
+
+Nginx locations can be find in `nginx.conf` file
+
+## Build
+
+Build all frontends on the main branch
+
+```sh
+docker build -t openpaas-front .
+```
+
+### Custom Build
+
+You can override standard build using docker build arguments
+
+#### Available build arguments
+
+---------------------------------------
+
+| Build Arg      | Description | Default Value |
+| ----------- | ----------- | ----------- |
+| NODE_VERSION             | NodeJS version | 12.19 |
+| NGINX_VERSION            | Nginx version | 1.19.3 |
+| INBOX_GIT_TREEISH        | Git treeish name for inbox frontend | main |
+| MAILTO_GIT_TREEISH       | Git treeish name for mailto frontend | main |
+| ACCOUNT_GIT_TREEISH      | Git treeish name for account frontend | main |
+| CONTACTS_GIT_TREEISH     | Git treeish name for contacts frontend | main |
+| CALENDAR_GIT_TREEISH     | Git treeish name for calendar frontend | main |
+| CALENDAR_PUB_GIT_TREEISH | Git treeish name for public calendar frontend | main |
```

Comment:
Duplicate argument

Reply:
?? which one

Reply:
`CALENDAR_PUB_GIT_TREEISH`

Reply:
CALENDAR_GIT_TREEISH   is different from CALENDAR_PUB_GIT_TREEISH  ?

Reply:
No I mean, you have a duplicate entry here 
![image](https://user-images.githubusercontent.com/7097550/97012523-37a01a00-1548-11eb-85e0-352293b1a583.png)

Reply:
you are right, good catch 👍